observatory_platform/cli/observatory.py

```diff
@@ -158,15 +158,15 @@ def get_env(dags_path: str, data_path: str, logs_path: str, postgres_path: str,
     env['FERNET_KEY'] = config.fernet_key
 
     # Set connections
-    env['AIRFLOW_CONN_MAG_RELEASES_TABLE'] = config.mag_releases_table_connection
-    env['AIRFLOW_CONN_MAG_SNAPSHOTS_CONTAINER'] = config.mag_snapshots_container_connection
-    env['AIRFLOW_CONN_CROSSREF'] = config.crossref_connection
+    env['AIRFLOW_CONN_MAG_RELEASES_TABLE'] = config.airflow_connections[AirflowConn.mag_releases_table.value]
+    env['AIRFLOW_CONN_MAG_SNAPSHOTS_CONTAINER'] = config.airflow_connections[AirflowConn.mag_snapshots_container.value]
+    env['AIRFLOW_CONN_CROSSREF'] = config.airflow_connections[AirflowConn.crossref.value]
 
     # Set variables
-    env['AIRFLOW_VAR_PROJECT_ID'] = config.project_id
+    env['AIRFLOW_VAR_PROJECT_ID'] = config.airflow_variables[AirflowVar.project_id.value]
     env['AIRFLOW_VAR_DATA_LOCATION'] = config.data_location
-    env['AIRFLOW_VAR_DOWNLOAD_BUCKET_NAME'] = config.download_bucket_name
-    env['AIRFLOW_VAR_TRANSFORM_BUCKET_NAME'] = config.transform_bucket_name
+    env['AIRFLOW_VAR_DOWNLOAD_BUCKET_NAME'] = config.airflow_variables[AirflowVar.download_bucket_name.value]
+    env['AIRFLOW_VAR_TRANSFORM_BUCKET_NAME'] = config.airflow_variables[AirflowVar.transform_bucket_name.value]
     env['AIRFLOW_VAR_ENVIRONMENT'] = config.environment.value
```

Comment:
Perhaps you could iterate over the airflow connections and variables here so that this section isn't hard coded too? Enums can be iterated: https://docs.python.org/3/library/enum.html#iteration and the name of the environment variable could be created with something like: `f'AIRFLOW_VAR_{enum_member.value.upper()}'`.

Reply:
That's a good idea, I've added this in the last commit (6e25630a5a0920b34081dcf433f85fde7e70881b) and made a couple of other small changes.